src/_nebari/constants.py

```diff
@@ -12,8 +12,8 @@
 DEFAULT_GKE_RELEASE_CHANNEL = "UNSPECIFIED"
 
 DEFAULT_NEBARI_DASK_VERSION = CURRENT_RELEASE
-DEFAULT_NEBARI_IMAGE_TAG = CURRENT_RELEASE
-DEFAULT_NEBARI_WORKFLOW_CONTROLLER_IMAGE_TAG = "2024.1.1"
+DEFAULT_NEBARI_IMAGE_TAG = "2024.2.1rc2"
```

Comment:
@dcmcand In https://github.com/nebari-dev/nebari/pull/2227 you did not have any changes for `DEFAULT_NEBARI_IMAGE_TAG` but I am assuming we do want to point to the latest RC in https://github.com/nebari-dev/nebari-docker-images, right? Otherwise we would be testing against older versions of the images.

Reply:
I specified them in the nebari-config.yaml rather than here. Bu this should be fine as long as we change it back to CURRENT_RELEASE before the release